Assignment2/adrix1341-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,61 @@
+{
+	"@context":{
+		"xsd": "http://www.w3.org/2001/XMLSchema#",
+		"person":"http://www.ontologies.org/ontologies/People#"

```

Comment:
You forgot defining the attributes.

Reply:
Still pending to add the attribute definitions into "@context" or add the corresponding prefix to them below

Reply:
I just made a change in the file. Sorry for taking time to answer